redditnav.js

```diff
@@ -71,26 +69,29 @@ function setUpButton($floatingButton, items) {
   $floatingButton.find('.mfb-component__button--main, .mfb-component__button--child').css('background-color', items.color);
   var buttonClass;
   switch(items.buttonPos){
-    case "right":
-      buttonClass = "mfb-component--br mfb-slidein-spring";
+    case 'right':
+      buttonClass = 'mfb-component--br mfb-slidein-spring';
       break;
-    case "left":
-      buttonClass = "mfb-component--bl mfb-slidein-spring";
+    case 'left':
+      buttonClass = 'mfb-component--bl mfb-slidein-spring';
       break;
-    case "hide":
-      buttonClass = "mfb-component--hide";
+    case 'hide':
+      buttonClass = 'mfb-component--hide';
       break;
   }
+
   $floatingButton.attr('class', buttonClass);
 }
 
 $(function() {
   chrome.storage.sync.get({
     color: '#FF5722',
     buttonPos: 'right'
-  }, function(items) {
```

Comment:
I personally like having the function expression starting on the same line to indicate that it's a callback.